.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Github Package Deploy
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  deployment:
+    name: Github Package Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install packages
+      run: |
+        sudo apt-get install libpq-dev -y
+        python3 -m pip install wheel twine setuptools poetry
```

Comment:
I don’t know for sure, but I think twine is alternative to poetry… it’s odd that both are required

Reply:
While on the subject, I think that poetry should be self-sufficient 🤔 
I can’t be totally sure, but I would not be surprised if wheel and setup tools are redundant here

Reply:
I tried running without wheel, twin and setuptools. Yes, seems like poetry is sufficient.